src/V3Randomize.cpp

```diff
@@ -1037,6 +1364,7 @@ void V3Randomize::randomizeNetlist(AstNetlist* nodep) {
     UINFO(2, __FUNCTION__ << ": " << endl);
     {
         const RandomizeMarkVisitor markVisitor{nodep};
+        V3Error::abortIfErrors();
```

Comment:
I prefer to keep error-exit aborts at top level; move this to Verilator.cpp right after it calls this visitor.  This is also so the dump below will occur.